docker.sh

```diff
@@ -158,7 +158,7 @@ for x in base web docs; do
 done
 
 # all files in subfolder
-for x in models executor data thredds monitor rstudio-nginx check; do
+for x in models executor data thredds monitor rstudio-nginx; do
```

Comment:
@robkooper there's a bunch of changes to Docker in this PR. Could you check to make sure these should be here and are not just the result of some sort of merge issue?